src/docs/user/references/Variables.adoc

```diff
@@ -141,22 +141,25 @@ Examples: PA:REGEXP([a-z]+) will accept "abc", "foo", but not "Foo".
+* *PA:CATALOG_OBJECT* :
 use this type when you want to reference and use, in the current workflow, another object from the Catalog. Various portals and tools will make it easy to manage catalog objects. For instance, at Job submission, you will be able to browse the Catalog to select the needed object. +
-Variable syntax must be a valid expression that matches the following pattern: `bucketName/objectName[/revision]`.
+Variable value syntax must be a valid expression that matches the following pattern: `bucketName/objectName[/revision]`.
 Note that the revision sub-pattern is a hash code number represented by 13 digit. +
-A filter can be specified in the PA:CATALOG_OBJECT model. In that case, the variable value must be a catalog object which matches the kind, content type, bucketName and objectName requirements. Note that kind and content type are case insensitive and require a "startsWith" matching, while bucketName and objectName are case sensitive and, by default, require a "contains" matching.
-That is, the kind and the content type of the provided catalog object must start with the filters specified in the model while bucketName and objectName of the catalog object should contain them.
-The scheduler server verifies that the object exists in the catalog and fulfills the requirements when the workflow is submitted. +
+Examples: PA:CATALOG_OBJECT will accept "bucket-example/object-example" and "bucket-example/object-example/1539310165443" but not "bucket-example/object-example/153931016" (invalid revision number) nor "bucket-example/" (missing object name).
+
+* *PA:CATALOG_OBJECT(kind,contentType,bucketName,objectName)* :
+One or more filters can be specified in the PA:CATALOG_OBJECT model to limit the accepted values. In that case, the variable value must be a catalog object which matches the `Kind`, `Content type`, `BucketName` and/or `ObjectName` requirements (For more information regarding Kind and Content type click <<_exportingimporting_objects_into_the_catalog, here>>). Note that Kind and Content type are case insensitive and require a "startsWith" matching, while BucketName and ObjectName are case sensitive and, by default, require a "contains" matching.
+That is, the Kind and the Content type of the provided catalog object must start with the filters specified in the model while BucketName and ObjectName of the catalog object should contain them.
+The scheduler server verifies that the object exists in the catalog and fulfills the specified requirements when the workflow is submitted. +
 Examples:
-** PA:CATALOG_OBJECT will accept "bucket-example/object-example/1539310165443" and "bucket-example/object-example" but not "bucket-example/object-example/153931016" (invalid revision number) and "bucket-example/" (missing object name).
+** PA:CATALOG_OBJECT(Workflow/standard) will accept standard workflow objects and not PSA workflows or scripts.
```

Comment:
- I think it's better to use the inclusive form of what should be used and not bother about mentioning what should not be used...

- Use the present tense only

```** PA:CATALOG_OBJECT(Workflow/standard) will accept standard workflow objects and not PSA workflows or scripts.```

-> 

```** PA:CATALOG_OBJECT(Workflow/standard) accepts only standard workflow objects.```

Reply:
I think mentioning what will not be accepted is interesting in our case because of the complexity of the filters and the different behaviours they have.

Ok for the present tense.